tests/unit/test_charm.py

```diff
@@ -1875,38 +1875,20 @@ def test_delete_password(harness, _has_secrets, caplog):
         assert harness.charm.get_secret("unit", "operator-password") is None
 
         harness.set_leader(True)
-        with caplog.at_level(logging.ERROR):
```

Comment:
removed this checking of error logs because it would break every time the lib gets updated and the log message is changed. Instead, we should only check if we can call `remove_secret` on non-existing secrets without error

Reply:
inspired by the approach of mysql: https://github.com/canonical/mysql-operator/pull/440

Reply:
Nononono.

Just change it to 
```
with caplog.at_level(logging.DEBUG):
```